### PR TITLE
fix(readme): add instructions to run unit and e2e tests

### PR DESCRIPTION
**Pull request by lorumic**
Added instructions to the README of the repo, to explain how to run unit tests and end-to-end tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,31 @@ Refer to the following examples to explore advanced usage scenarios and best pra
...
 * [JavaScript-based Streaming Example](examples/js) - A web application utilizes the JS SDK for video streaming.
 
+## Testing
+
+All tests (unit tests with [Jest](https://jestjs.io) and end-to-end tests with [Playwright](https://playwright.dev)) run inside a docker container, so docker must be installed.
+
+- Unit tests are located in `js/tests/unit`.
+- End-to-end tests are located in `js/tests/e2e`.
+
+Both these folders contain a `run-tests.sh` script. You just need to run this script to start the tests. However, to run end-to-end tests you need also an Anbox Cloud back-end. The easiest way to set up one is by using the [Anbox Cloud Appliance](https://documentation.ubuntu.com/anbox-cloud/en/latest/tutorial/installing-appliance/).
```
Comment (keirthana) [inline README.md:35]:
```suggestion
Both these folders contain a `run-tests.sh` script that you can run to start the tests. However, to run end-to-end tests, you also need an Anbox Cloud deployment. The easiest way to set up one is by using the [Anbox Cloud Appliance](https://documentation.ubuntu.com/anbox-cloud/en/latest/tutorial/installing-appliance/).
```

File: README.md [hunk 2]
```diff
@@ -25,6 +25,31 @@ Refer to the following examples to explore advanced usage scenarios and best pra
...
+
+Once the back-end is ready, follow these steps to run the end-to-end tests:
```
Comment (keirthana) [inline README.md:37]:
```suggestion
Once the appliance is ready, follow these steps to run the end-to-end tests:
```

**Review by keirthana — CHANGES_REQUESTED** [review]
Some minor comments

**Comment by lorumic** [comment]
Thanks for the great suggestions @keirthana - I have addressed your comments [here](https://github.com/canonical/anbox-streaming-sdk/compare/57bec1d9c35c8852eda6d5d7ced374f9bc8d3970..75a5505971419f199fe7150aad464371f34ed4f7).

**Review by morphis — APPROVED** [review]
LGTM

**Review by adglkh — APPROVED** [review]
LGTM

**Review by ajanon — APPROVED** [review]
LGTM

**Review by keirthana — APPROVED** [review]
Thanks, LGTM

**Pull request merged**